src/livecodes/index.ts

```diff
@@ -107,18 +106,4 @@ window.addEventListener(customEvents.destroy, () => {
   document.head.innerHTML = '';
 });
 
-if (isEmbed && params.get('config') === 'sdk') {
-  addEventListener(
-    'message',
-    function configHandler(
-      e: MessageEventInit<{ type: CustomEvents['config']; payload: Partial<Config> }>,
-    ) {
-      if (e.source !== parent || e.data?.type !== customEvents.config) return;
-      removeEventListener('message', configHandler);
-      livecodes('#livecodes', e.data.payload).then(loaded);
-    },
-  );
-  parent.postMessage({ type: customEvents.getConfig }, '*');
-} else {
-  livecodes('#livecodes').then(loaded);
-}
+livecodes('#livecodes').then(loaded);
```

Comment:
I agree with your initial suggestion.
Let's revert this as it was for backward-compatibility.
Sorry!